### PR TITLE
Pytorch main

### PR DESCRIPTION
**Pull request by ekinakyurek**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [ ] fix a bug
- [ ] update tests and/or documentation
- [ ] other (please add here)

Please link to any issues this PR addresses.

#### Changelog
What are the changes made in this PR?

#### Test plan
Please make sure to do each of the following if applicable to your PR. (If you're not sure about any one of these just ask and we will happily help.)

- [ ] run pre-commit hooks and linters (make sure you've first installed via `pre-commit install`)
- [ ] add unit tests for any new functionality
- [ ] update docstrings for any new or updated methods or classes
- [ ] run unit tests via `pytest tests`
- [ ] run recipe tests via `pytest tests -m integration_test`
- [ ] manually run any new or modified recipes with sufficient proof of correctness
	- [ ] include relevant commands and any other artifacts in this summary (pastes of loss curves, eval results, etc.)

### CODE REVIEW

File: recipes/configs/llama3/8B_full.yaml [hunk 1]
```diff
@@ -21,43 +21,52 @@
 # Tokenizer
 tokenizer:
   _component_: torchtune.models.llama3.llama3_tokenizer
-  path: /tmp/Meta-Llama-3-8B-Instruct/original/tokenizer.model
+  path: /raid/lingo/models/Meta-Llama-3-8B-Instruct/original/tokenizer.model
```
Comment (SalmanMohammadi) [inline recipes/configs/llama3/8B_full.yaml:24]: looks like you did some fine-tuning? : )

Keeping yaml configs as-default in the repo would help ensure consistency with docs and ensures there's exemplar config files around. I've found `tune cp` helpful to create local copies of configs from the repo.

File: torchtune/utils/_checkpointing/_checkpointer_utils.py [hunk 2]
```diff
@@ -75,6 +75,7 @@ def safe_torch_load(checkpoint_path: Path, weights_only: bool = True) -> Dict[st
                 weights_only=weights_only,
             )
     except Exception as e:
+        breakpoint()
```
Comment (SalmanMohammadi) [inline torchtune/utils/_checkpointing/_checkpointer_utils.py:78]: some debugging artefact?

**Comment by ekinakyurek** [comment]
I am not sure how I opened the pull request here, I wanted to work on my own fork :) I am closing immediately.
